jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamo/editor/FiltersList.kt

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamo.editor
+
+import com.intellij.ui.ToolbarDecorator
+import com.intellij.ui.table.JBTable
+import com.intellij.util.ui.ListTableModel
+
+class FiltersList {
+    data class Filter(val attributeName: String)
+
+    private val table = JBTable(ListTableModel<Filter>())
+    private val toolbar = ToolbarDecorator.createDecorator(table)
+
+    init {
+        table.emptyText.text = "No filters applied"
```

Comment:
localize

Reply:
Whopps, filters list wasnt supposed to live on in this branch